rpkg_src/import_rpkg_meta.cpp

```diff
@@ -182,17 +178,15 @@ bool rpkg_function::import_rpkg_meta(rpkg& rpkg_meta_data, std::string& rpkg_met
 
         return true;
     }
-    else
-    {
-        rpkg_meta_data.is_patch_file = true;
-        rpkg_meta_data.patch_entry_count = 0x0;
 
-        LOG("RPKG meta file matching the name of the folder was not found");
-        LOG("   in it's root:" << input_rpkg_folder_path);
-        LOG("Defaulting to RPKGv1 patch file creation settings:");
-        LOG("  - Patch entry (deletion list) count is set to 0");
-        LOG("  - No patch entries (deletion entries) will be appended");
+    rpkg_meta_data.is_patch_file = true;
+    rpkg_meta_data.patch_entry_count = 0x0;
 
-        return false;
-    }
-}
\ No newline at end of file
+    LOG("RPKG meta file matching the name of the folder was not found");
+    LOG("   in it's root:" << input_rpkg_folder_path);
+    LOG("Defaulting to RPKGv1 patch file creation settings:");
+    LOG("  - Patch entry (deletion list) count is set to 0");
+    LOG("  - No patch entries (deletion entries) will be appended");
+
+    return false;
+}
```

Comment:
The else branch here is useless because the previous branch will always return.

Reply:
There are if statements like this littered around the codebase.

Reply:
If a big refactor is done before a rewrite (rewrite is definitely more preferential), these can be addressed. But right now they stay like this. They'll be optimised by the compiler.

Reply:
I'm working on at least trying to rewrite it, but in order to do so, I'd prefer a cleaner codebase to work with.